kythe/go/extractors/config/extractor.go

```diff
@@ -196,24 +197,22 @@ func verifyRequiredTools() error {
 }
 
 func findConfig(configPath, repoDir string) (*ecpb.ExtractionConfiguration, error) {
+	var configReader io.Reader
```

Comment:
I guess we could probably simplify this

```
	if configPath == "" {
		return Load(mvn.DefaultConfig())
	}
	f, err := os.Open(configPath)
	if os.IsNotExist(err) {
		return Load(mvn.DefaultConfig())
	} else if err != nil {
		return nil, fmt.Errorf("opening config file: %v", err)
	}
	defer f.Close()
	return Load(f)
```

Reply:
Hmm maybe but that's not the same logic.

If configPath is not passed, I still want to check the repo itself.  If the repo doesn't have it, then fall back.

But I agree we could just return Load(), and multiple calls to Load allows us to drop the var thingy (and more properly Close the file)